src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo | null;
+  setVisible: React.Dispatch<React.SetStateAction<boolean>>;
+  setTodo: React.Dispatch<React.SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({ setVisible, todo, setTodo }) => {
+  const [isLoad, setIsLoad] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setIsLoad(false);
+    }, 1000);
+
+    if (todo) {
+      getUser(todo.userId).then(setUser);
+    }
+  }, []);
+
+  if (!todo) {
+    return;
```

Comment:
The `return` statement here does not return any JSX or null, which may lead to unexpected behavior. Consider returning `null` explicitly if the `todo` is not available.